frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/prefill-vm-template-state-update.ts

```diff
@@ -158,41 +162,48 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
     const volumeLookup = createBasicLookup<V1Volume>(getVolumes(vm), getSimpleName);
     const datavolumeTemplatesLookup = createBasicLookup(getDataVolumeTemplates(vm), getName);
     // // prefill storage
-    const templateStorages: VMWizardStorage[] = getDisks(vm).map((disk) => {
-      const diskWrapper = DiskWrapper.initialize(disk);
-      let volume = volumeLookup[diskWrapper.getName()];
-      const volumeWrapper = VolumeWrapper.initialize(volume);
-
-      if (volumeWrapper.getType() === VolumeType.CLOUD_INIT_NO_CLOUD) {
-        const helper = new CloudInitDataHelper(volumeWrapper.getCloudInitNoCloud());
-        if (helper.includesOnlyFormValues()) {
-          isCloudInitForm = true;
-          helper.makeFormCompliant();
-          volume = new MutableVolumeWrapper(volume, { copy: true })
-            .replaceTypeData(helper.asCloudInitNoCloudSource())
-            .asMutableResource();
-          // do not overwrite with more cloud-init disks
-        } else if (isCloudInitForm == null) {
-          isCloudInitForm = false;
+    const templateStorages: VMWizardStorage[] = getDisks(vm)
+      .filter(
+        (disk) => !(DiskWrapper.initialize(disk).getName() === WINTOOLS_DISK_NAME && !isWindows),
```

Comment:
I do not think it is wise to override the templates - they might have different values for a reason.